system-lock-formula/metadata/form.yml

```diff
@@ -2,12 +2,12 @@ minion_blackout:
   $type: boolean
   $default: False
   $name: Lock system
-  $help: Disable all remote execution commands
+  $help: Disable all actions that can be issued on the minion
```

Comment:
All, or "except the ones configured in the configuration of this formula"?

Reply:
or (shorter) "almost" 😸